[All hosts](samples) Add hello world samples (#3094)
OfficeDev/office-js-docs-pr: docs/overview/office-add-in-code-samples.md

---
title: Office Add-in code samples
description: 'A list of Office Add-in code samples to help you learn and build your own add-ins.'
ms.date: 11/18/2021
localization_priority: high
---

# Office Add-in code samples

These code samples are written to help you learn how to use various features when developing Office Add-ins.

## Getting started

> The following samples show how to build the simplest Office Add-in with only a manifest, HTML web page, and a logo. They will help you understand the fundamental parts of an Office Add-in. For additional getting started information, see our [quick starts](../quickstarts/excel-quickstart-jquery.md) and [tutorials](/search/?terms=tutorial&scope=Office%20Add-ins).

Reviewer: Quick suggestion to help the second sentence sound less repetitive. 
```suggestion
The following samples show how to build the simplest Office Add-in with only a manifest, HTML web page, and a logo. These components are the fundamental parts of an Office Add-in. For additional getting started information, see our [quick starts](../quickstarts/excel-quickstart-jquery.md) and [tutorials](/search/?terms=tutorial&scope=Office%20Add-ins).
```
